tests/customerSite/menu/viewCoffee/coffeeCupHasCorrectCost.spec.js

```
@@ -1,19 +1,35 @@
 import { test } from '../../../_fixtures/fixtures';
+import { allure } from 'allure-playwright';
+
 import { priceFormatStr } from '../../../../src/common/helpers/priceFormatters';
 import { COFFEE_NAMES, COFFEE_PRICES } from '../../../../src/constants';
 
-let testParameters = [];
+const testParameters = Object.entries(COFFEE_NAMES).map(([key, name]) => ({
+  coffee: name,
+  price: COFFEE_PRICES[key],
+}));
 
-for (const [key, value] of Object.entries(COFFEE_NAMES)) {
-  testParameters.push({ coffee: value, price: COFFEE_PRICES[key] });
-}
+test.describe('Menu > View Coffee', () => {
+  // Shared Allure labels
+  test.beforeEach(async () => {
+    await allure.parentSuite('Customer site');
+    await allure.suite('Menu');
+    await allure.subSuite('View Coffee');
+    await allure.epic('CoffeeCart Customer site');
+    await allure.feature('Menu');
+    await allure.severity('minor');
```

In this file's `beforeEach` you call `await allure.parentSuite(...)`, `allure.suite(...)`, `allure.subSuite(...)`, `allure.epic(...)`, `allure.feature(...)`, `allure.severity(...)` using the module-level `allure`. This will not reliably attach labels to the current test and conflicts with the requirement to use the Playwright adapter API.

Change to the per-test adapter and ensure you fail fast when it's not available (see earlier comment for suggested check).